### PR TITLE
Feat: like & dislike

### PR DESCRIPTION
**Pull request by anwari666**
Merge after #22 please, otherwise it might raise conflict

### CODE REVIEW

File: src/Home.js [hunk 1]
```diff
@@ -29,25 +30,61 @@ function Home () {
...
+
     if ( shouldFetch() && ( !isFetching || !isFetchingNextPage ) ) {
-      console.log(`GET MORE GIRLS! Seen: ${allGirls.length}, swiped: ${swipedGirls.current.length}.`)
+      console.log(`GET MORE GIRLS! Total: ${allGirls.length}, removed: ${removedGirls.current.length}.`)
       fetchNextPage()
     }
   }
 
-  const shouldFetch = () => (allGirls.length - swipedGirls.current.length) === REFETCH_THRESHOLD
+  const swipe = async (dir) => {
```
Comment (ans-4175) [inline src/Home.js:54]: ooh implementasi fungsi swipe-nya

File: src/Home.js [hunk 2]
```diff
@@ -29,25 +30,61 @@ function Home () {
   
   const swiped = (direction, id) => {
     const girl = allGirls.find((item) => item.id === id);
-    console.log('removing', girl);
+    console.log('swiping ', girl);
+    !swipedGirls.current.includes(id) && swipedGirls.current.push(id)
```
Comment (ans-4175) [inline src/Home.js:34]: oh nice dipastikan dulu

File: src/Home.js [hunk 3]
```diff
@@ -29,25 +30,61 @@ function Home () {
...
-      console.log(`GET MORE GIRLS! Seen: ${allGirls.length}, swiped: ${swipedGirls.current.length}.`)
+      console.log(`GET MORE GIRLS! Total: ${allGirls.length}, removed: ${removedGirls.current.length}.`)
       fetchNextPage()
     }
   }
 
-  const shouldFetch = () => (allGirls.length - swipedGirls.current.length) === REFETCH_THRESHOLD
+  const swipe = async (dir) => {
+    const currentGirl = getCurrentlyShownGirl()
+    !!currentGirl && childRefs.current[currentGirl.id].current.swipe(dir)
```
Comment (ans-4175) [inline src/Home.js:56]: dia bakal trigger `swiped` nya gimana sih?

File: src/Home.js (hunk 2 above)
Comment (anwari666) [reply src/Home.js:34]: iya, ada kasus tadi kl ngekliknya semangat, ternyata list nya habis duluan jadi si `id`nya `undefined`

File: src/Home.js (hunk 3 above)
Comment (anwari666) [reply src/Home.js:56]: Line 56 itu shorthand, semacam if `currentGirl` nya truthy, then `childRefs.current[currentGirl.id].current`nya eksekusi method `.swipe()`

**Review by ans-4175 — APPROVED** [review]
Approved #24 will be merged after #22

**Pull request merged**